### ISSUE TITLE
Extension not buildable

### ISSUE THREAD

**Issue by chillyistkult** [passbolt/passbolt_browser_extension#144]
Extension is not buildable with node / npm  8.1.0. 

```
npm ERR! code 1
npm ERR! git dep preparation failed
npm ERR! command /usr/local/Cellar/node/17.0.1/bin/node /usr/local/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/Users/foo/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit --include=dev --include=peer --include=optional --no-package-lock-only --no-dry-run
npm ERR! > i18next-scanner@3.0.0 prepublish
npm ERR! > npm run eslint && npm run build && npm test
npm ERR! 
npm ERR! 
npm ERR! > i18next-scanner@3.0.0 eslint
npm ERR! > eslint ./src
npm ERR! 
npm ERR! 
npm ERR! > i18next-scanner@3.0.0 build
npm ERR! > babel ./src --out-dir ./lib
npm ERR! 
npm ERR! Successfully compiled 6 files with Babel (1076ms).
npm ERR! 
npm ERR! > i18next-scanner@3.0.0 test
npm ERR! > tap test/*.js --no-esm --no-timeout --node-arg=--require --node-arg=@babel/register --node-arg=--require --node-arg=core-js/stable --node-arg=--require --node-arg=regenerator-runtime/runtime
npm ERR! 
npm ERR! TAP version 13
npm ERR! ok 1 - test/jsx-parser.js # time=76.914ms {
npm ERR! ...
npm ERR! 1..3
npm ERR! # failed 2 of 3 tests
npm ERR! ------------------------|----------|----------|----------|----------|-------------------|
npm ERR! File                    |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
npm ERR! ------------------------|----------|----------|----------|----------|-------------------|
npm ERR! All files               |     89.2 |    84.33 |    93.33 |    88.65 |                   |
npm ERR!  acorn-jsx-walk.js      |      100 |       50 |    85.71 |      100 |              7,21 |
npm ERR!  flatten-object-keys.js |      100 |       90 |      100 |      100 |                22 |
npm ERR!  index.js               |    98.44 |      100 |      100 |    98.39 |                96 |
npm ERR!  nodes-to-string.js     |    81.82 |    73.53 |      100 |     81.4 |... 29,54,60,61,65 |
npm ERR!  omit-empty-object.js   |      100 |    83.33 |      100 |      100 |                25 |
npm ERR!  parser.js              |    87.72 |    84.55 |    91.84 |    86.97 |... 94,996,997,999 |
npm ERR! ------------------------|----------|----------|----------|----------|-------------------|
npm ERR! npm WARN using --force Recommended protections disabled.
npm ERR! npm WARN ERESOLVE overriding peer dependency
npm ERR! npm WARN While resolving: acorn-dynamic-import@4.0.0
npm ERR! npm WARN Found: acorn@8.7.0
npm ERR! npm WARN node_modules/acorn
npm ERR! npm WARN   acorn@"^8.0.4" from the root project
npm ERR! npm WARN   2 more (acorn-jsx, acorn-stage3)
npm ERR! npm WARN 
npm ERR! npm WARN Could not resolve dependency:
npm ERR! npm WARN peer acorn@"^6.0.0" from acorn-dynamic-import@4.0.0
npm ERR! npm WARN node_modules/acorn-dynamic-import
npm ERR! npm WARN   acorn-dynamic-import@"^4.0.0" from the root project
npm ERR! npm WARN 
npm ERR! npm WARN Conflicting peer dependency: acorn@6.4.2
npm ERR! npm WARN node_modules/acorn
npm ERR! npm WARN   peer acorn@"^6.0.0" from acorn-dynamic-import@4.0.0
npm ERR! npm WARN   node_modules/acorn-dynamic-import
npm ERR! npm WARN     acorn-dynamic-import@"^4.0.0" from the root project
npm ERR! npm WARN ERESOLVE overriding peer dependency
npm ERR! npm WARN While resolving: eslint-config-trendmicro@1.4.1
npm ERR! npm WARN Found: eslint@7.32.0
npm ERR! npm WARN node_modules/eslint
npm ERR! npm WARN   dev eslint@"^7.13.0" from the root project
npm ERR! npm WARN   4 more (babel-eslint, eslint-plugin-import, ...)
npm ERR! npm WARN 
npm ERR! npm WARN Could not resolve dependency:
npm ERR! npm WARN peer eslint@"^5.2.0" from eslint-config-trendmicro@1.4.1
npm ERR! npm WARN node_modules/eslint-config-trendmicro
npm ERR! npm WARN   dev eslint-config-trendmicro@"^1.4.1" from the root project
npm ERR! npm WARN 
npm ERR! npm WARN Conflicting peer dependency: eslint@5.16.0
npm ERR! npm WARN node_modules/eslint
npm ERR! npm WARN   peer eslint@"^5.2.0" from eslint-config-trendmicro@1.4.1
npm ERR! npm WARN   node_modules/eslint-config-trendmicro
npm ERR! npm WARN     dev eslint-config-trendmicro@"^1.4.1" from the root project
npm ERR! npm WARN ERESOLVE overriding peer dependency
npm ERR! npm WARN ERESOLVE overriding peer dependency
npm ERR! npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm ERR! npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm ERR! npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm ERR! npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm ERR! npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm ERR! npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
npm ERR! npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm ERR! npm WARN deprecated chokidar@2.1.8: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies
npm ERR! npm WARN deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
npm ERR! npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm ERR! npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! npm WARN using --force Recommended protections disabled.
npm ERR! npm WARN using --force Recommended protections disabled.
npm ERR! npm WARN using --force Recommended protections disabled.
npm ERR! i18next-scanner: "none" does not exist in the namespaces (["translation"]): key="key2", options={}
npm ERR! (node:20176) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
npm ERR! (Use `node --trace-deprecation ...` to show where the warning was created)
npm ERR! npm ERR! code 1
npm ERR! npm ERR! path /Users/foo/.npm/_cacache/tmp/git-clonegWqlgu
npm ERR! npm ERR! command failed
npm ERR! npm ERR! command sh -c npm run eslint && npm run build && npm test
npm ERR! 

```

Please fix.

**Comment by stripthis** [comment]
Thanks for the report, we'll look into it.

**Comment by chillyistkult** [comment]
I tried with `yarn install` and it worked. Most likely it would also work with some older node version, would be at least nice to indicate which node version is needed in `package.json`.

**Comment by cedricalfonsi** [comment]
@chillyistkult It should have been solved with the v3.5.2.

Feel free to reopen if you are still encounterring the issue.
